ldi-orchestrator/ldio-connectors/ldio-kakfa-in/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaInAutoConfig.java

```diff
@@ -61,16 +57,40 @@ private String getContentType(ComponentProperties config) {
 					.orElse(Lang.NQUADS.getHeaderString());
 		}
 
-		private Map<String, Object> consumerProps(String bootstrapServers, String groupId, String offsetReset) {
-			Map<String, Object> props = new HashMap<>();
-			props.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapServers);
-			props.put(ConsumerConfig.GROUP_ID_CONFIG, groupId);
-			props.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, offsetReset);
+		private Map<String, Object> getConsumerConfig(ComponentProperties config) {
+			final Map<String, Object> props = new HashMap<>();
```

Comment:
I don't think we ever use "final" in methods.. can you reason why you would want it here?

Reply:
I always make variables final to discourage people from overriding variables and indicate intent. It's not that uncommon: https://softwareengineering.stackexchange.com/questions/115690/why-declare-final-variables-inside-methods